proc_scripts/load_data.py

```diff
@@ -0,0 +1,139 @@
+    print(s.getaxis('indirect'))
+    print(s.getaxis('ph1'))
+    print(s.getaxis('ph2'))
+    s.setaxis('ph1',r_[0:4.]/4) #setting values of axis ph1 to line up
+    s.setaxis('ph2',r_[0:2.]/4) #setting values of axis ph1 to line up
+    s.setaxis('indirect', s.get_prop('vd'))
+    fl.next('time domain') #switch to time domain as a string based name for fig
+    fl.image(s) #labeling
+    fl.next('FT + coherence domain')
+#titling to coherence domain
+    s.ft('t2',shift=True) #fourier transform
+    s.ft(['ph1','ph2']) #fourier transforming from phase cycle dim to coherence dimension
+    s.reorder(['indirect','t2'], first=False)
+    print("after reorder",ndshape(s))
+    fl.image(s)
+    return s
+    #raise RuntimeError("this is where postprocessing would be implemented -- not implemented yet")
+
+def proc_CPMG(s):
+    print("loading pre-processing for CPMG preprocessing")
+    SW_kHz = s.get_prop('acq_params')['SW_kHz']
+    nPoints = s.get_prop('acq_params')['nPoints']
+    nEchoes = s.get_prop('acq_params')['nEchoes']
+    nPhaseSteps = s.get_prop('acq_params')['nPhaseSteps']
+    nScans = s.get_prop('acq_params')['nScans']
+    p90_s = s.get_prop('acq_params')['p90_us']*1e-6
+    deadtime_s = s.get_prop('acq_params')['deadtime_us']*1e-6
+    deblank_s = s.get_prop('acq_params')['deblank_us']*1e-6
+    marker_s = s.get_prop('acq_params')['marker_us']*1e-6
+    tau1_s = s.get_prop('acq_params')['tau1_us']*1e-6
+    pad_start_s = s.get_prop('acq_params')['pad_start_us']*1e-6
+    pad_end_s = s.get_prop('acq_params')['pad_end_us']*1e-6
+    orig_t = s.getaxis('t')
+    acq_time_s = orig_t[nPoints]
+    s.set_units('t','s')
+    twice_tau = deblank_s + 2*p90_s + deadtime_s + pad_start_s + acq_time_s + pad_end_s + marker_s
+    t2_axis = linspace(0,acq_time_s,nPoints)
+    tE_axis = r_[1:nEchoes+1]*twice_tau
+    s.setaxis('t',None)
+    s.setaxis('nScans',r_[0:nScans])
+    s.chunk('t',['ph1','tE','t2'],[nPhaseSteps,nEchoes,-1])
+    s.setaxis('ph1',r_[0.,2.]/4)
+    s.setaxis('tE',tE_axis)
+    s.setaxis('t2',t2_axis)
+    s.ft('t2', shift=True)
+    return s
+
+def proc_Hahn_echoph(s):
+    print("loading pre-processing for Hahn_echoph")
+    fl = figlist_var()
+    nPoints = s.get_prop('acq_params')['nPoints']
+    nEchoes = s.get_prop('acq_params')['nEchoes']
+    nPhaseSteps = 8 
+    SW_kHz = s.get_prop('acq_params')['SW_kHz']
+    nScans = s.get_prop('acq_params')['nScans']
+    s.reorder('t',first=True)
+    t2_axis = s.getaxis('t')[0:256]
+    s.setaxis('t',None)
+    s.chunk('t',['ph2','ph1','t2'],[2,4,-1])
+    s.labels({'ph2':r_[0.,2.]/4,
+        'ph1':r_[0.,1.,2.,3.]/4})
+    s.reorder(['ph2','ph1'])
+    #s.setaxis('ph2',r_[0.,2.]/4)
+    #s.setaxis('ph1',r_[0.,1.,2.,3.]/4)
```

Comment:
delete

Reply:
done

Reply:
so, actually, you don't need to go through and say "done", I realized -- once you update, the code shows up as "outdated" in the top right (meaning that my comment is outdated -- that the code looks different than it did when I made the comment), and if I want to, I can check to see whether or not it's changed in a way that makes sense.

actually, right now, I can see right away you haven't pushed these changes, b/c there is not "outdated" anymore